src/Browser/AuthController.js

```diff
@@ -190,7 +190,16 @@ class AuthController {
   * @private
   */
   async getAccess() {
-    const res = await utils.superagent.get(this.accessData.poll).set('accept', 'json');
+    let res;
+    try{
+      res = await utils.superagent.get(this.accessData.poll).set('accept', 'json');
+    } 
+    catch(e){
+      const body  = {
+        "status" : 'ERROR'
+      }
+      return body
```

Comment:
It would be more clear to write what's in the catch like this:
`return { "status" : 'ERROR' }` or `res.body =  { "status" : 'ERROR' }`

Reply:
Thank you for the review. I fixed the points you mention.

Reply:
All right. Fine for me to merge then